internal/cloudmock/metrics.go

```diff
@@ -104,10 +105,16 @@ func (f *fakeMetricServiceServer) CreateTimeSeries(
 	var code codes.Code
 	if strings.Contains(req.Name, "notfound") {
 		code = codes.NotFound
+	} else if strings.Contains(req.Name, "unavailable") {
```

Comment:
Please add docs somewhere denoting these magic strings.

Reply:
These are only used in this internal mock server for testing. But I added a comment for this function explaining that a lot better